tests/js/ttimes.nim

```diff
@@ -9,21 +8,32 @@ import times
 # Tue 19 Jan 03:14:07 GMT 2038
 
 block yeardayTest:
-  # check if yearday attribute is properly set on TimeInfo creation
-  doAssert fromSeconds(2147483647).getGMTime().yearday == 18
-
-block localTimezoneTest:
-  # check if timezone is properly set during Time to TimeInfo conversion
-  doAssert fromSeconds(2147483647).getLocalTime().timezone == getTimezone()
-
-block timestampPersistenceTest:
-  # check if timestamp persists during TimeInfo to Time conversion
-  const
-    timeString = "2017-03-21T12:34:56+03:00"
-    timeStringGmt = "2017-03-21T09:34:56+00:00"
-    timeStringGmt2 = "2017-03-21T08:34:56+00:00"
-    fmt = "yyyy-MM-dd'T'HH:mm:sszzz"
-  # XXX Check which one is the right solution here:
-
-  let x = $timeString.parse(fmt).toTime().getGMTime()
-  doAssert x == timeStringGmt or x == timeStringGmt2
+  doAssert fromUnix(2147483647).utc.yearday == 18
+
+block localTime:
+  var local = now()
+  let utc = local.utc
+  doAssert local.toTime.toSeconds == utc.toTime.toSeconds
+
+let a = fromUnix(1_000_000_000)
+let b = fromUnix(1_500_000_000)
+discard a - b
+
+# Because we can't change the timezone JS use a simple static timezone for testing.
```

Comment:
In the old code the label gave a hint as to what was being tested: ``timestampPersistanceTest``. In your code I see no clues like this. Please add more comments.